Source/Scene/Cesium3DTileset.js

```diff
@@ -1502,10 +1497,7 @@ define([
                 tileset._statistics.incrementLoadCounts(tile.content);
                 ++tileset._statistics.numberOfTilesWithContentReady;
 
-                // Add to the tile cache. Previously expired tiles are already in the cache.
```

Comment:
Should this doc be kept since checks on `add` are abstracted away now?

Reply:
Ah yes - fixed.